Source/WebCore/loader/archive/cf/LegacyWebArchive.cpp

```diff
@@ -62,6 +62,9 @@
 
 namespace WebCore {
 
+static constexpr unsigned maxFileNameSizeInBytes = 255;
+static constexpr char escapeCharactersForFileName[] = ":/";
```

Comment:
Hm... do we really need to have a hard-coded list like this?

Reply:
🤔Could you be more specific about "this"? Do you think it should not be a list or should not be global?

Reply:
I tested more special names and it seems the other browsers do escape special characters by replacing them with ' ' or '_'. I previously said they don't as they don't encode the one I tested.  To match the escaping behavior, in the latest patch, I remove the hard-coded list and use `FileSystem::encodeForFileName` instead. Thanks for the suggestion. 
I also update the test to cover that.